_limitations/system_function_limitations.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Unsupported Functions, Views, and Procedures 
+nav_order: 3
+---
+
+## Unsupported functions, views, and procedures
+
+Babelfish provides limited support for SQL Server system-implemented functions, views, and procedures. The following lists detail the functionality implemented for each version of Babelfish (and subsequent releases).
+
+### Built-in Functions
+
+The following Built-in Functions are supported for versions 2.1.0 and higher: 
+
+- IS_MEMBER()
+- IS_ROLEMEMBER()
+- HAS_PERMS_BY_NAME() 
+
+The following Built-in functions are supported for versions 1.2.0 and higher: 
+
+- COLUMNS_UPDATED()
+- UPDATE()
+- FULLTEXTSERVICEPROPERTY()
+- ISJSON()
+- JSON_QUERY()
+- JSON_VALUE()
+- HAS_DBACCESS()
+- SUSER_SID()
+- SUSER_SNAME()
+- IS_SRVROLEMEMBER()
+
+The following Built-in functions are supported for versions 1.0.0 and higher:
+
+- `SQUARE()`
+- `CONCAT_WS()`
+- `DATEFROMPARTS()`
+- `DATETIMEFROMPARTS()`
+- `ORIGINAL_LOGIN()`
+- `SESSION_USER()`
+- `CHOOSE()`
+- `CURSOR_STATUS()`
+- `SCHEMA_NAME()`
+- `TRIGGER_NESTLEVEL()` is supported but only without arguments.
+
+### System Catalog Views
+
+The following System Catalog Views are supported for versions 2.1.0 and higher: 
+
+- `sys.syslanguages`
+- `sys.indexes`
+- `sys.all_views`
+- `sys.database_files`
+- `sys.sql_modules`
+- `sys.system_sql_modules`
+- `sys.all_sql_modules`
+- `sys.xml_schema_collections`
+- `sys.dm_hadr_database_replica_states`
+- `sys.data_spaces`
+- `sys.database_mirroring`
+- `sys.database_role_members` 
+
+The following System Catalog Views are supported for versions 1.2.0 and higher:
```

Comment:
1.1.0 and 1.0.0 are missing?

Reply:
@rcv-aws Can you provide a list of unsupported catalog views for each version by any chance?

Reply:
Thank you!!!  This problem be addressed in a subsequent PR to avoid merge conflicts...